ballerina/client.bal

```diff
@@ -0,0 +1,743 @@
+    # + payload - Data required to update the calendar.
+    # + return - Successful response 
+    resource isolated function patch calendars/[string calendarId](Calendar payload, "json"? alt = (), string? fields = (), string? 'key = (), string? oauth_token = (), boolean? prettyPrint = (), string? quotaUser = ()) returns Calendar|error {
+        string resourcePath = string `/calendars/${getEncodedUri(calendarId)}`;
+        map<anydata> queryParam = {"alt": alt, "fields": fields, "key": 'key, "oauth_token": oauth_token, "prettyPrint": prettyPrint, "quotaUser": quotaUser};
+        resourcePath = resourcePath + check getPathForQueryParam(queryParam);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        Calendar response = check self.clientEp->patch(resourcePath, request);
+        return response;
+    }
+
+    # Returns the rules in the access control list for the calendar.
+    #
+    # + alt - Data format for the response.
+    # + fields - Selector specifying which fields to include in a partial response.
+    # + 'key - API key. Your API key identifies your project and provides you with API access, quota, and reports. Required unless you provide an OAuth 2.0 token.
+    # + oauth_token - OAuth 2.0 token for the current user.
+    # + prettyPrint - Returns response with indentations and line breaks.
+    # + quotaUser - An opaque string that represents a user for quota purposes. Must not exceed 40 characters.
+    # + calendarId - Calendar identifier. To retrieve calendar IDs call the calendarList.list method. If you want to access the primary calendar of the currently logged in user, use the "primary" keyword.
+    # + maxResults - Maximum number of entries returned on one result page. By default the value is 100 entries. The page size can never be larger than 250 entries. Optional.
+    # + pageToken - Token specifying which result page to return. Optional.
+    # + showDeleted - Whether to include deleted ACLs in the result. Deleted ACLs are represented by role equal to "none". Deleted ACLs will always be included if syncToken is provided. Optional. The default is False.
+    # + syncToken - Token obtained from the nextSyncToken field returned on the last page of results from the previous list request. It makes the result of this list request contain only entries that have changed since then. All entries deleted since the previous list request will always be in the result set and it is not allowed to set showDeleted to False.
+    # If the syncToken expires, the server will respond with a 410 GONE response code and the client should clear its storage and perform a full synchronization without any syncToken.
+    # Learn more about incremental synchronization.
+    # Optional. The default is to return all entries.
+    # + return - Successful response 
```

Comment:
```suggestion
    # + return - Successful response 
```
Shall we improve the return statements

Reply:
Fixed in [ed81f7c](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/134/commits/ed81f7cb7f43ae127ccbee9ef1cade8db0cb7658)